Removed incorrect vSphere link and replaced step (#83887)
openshift/openshift-docs: modules/update-vsphere-virtual-hardware-on-template.adoc

Once converted from a template, do not power on the virtual machine.
====

> . Update the virtual machine (VM) in the vSphere client. Follow link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-60768C2F-72E1-42E0-8A17-CA76849F2950.html[Upgrade the Compatibility of a Virtual Machine Manually] in the VMware documentation for more information.

Reviewer: ```suggestion
. Update the virtual machine (VM) in the {vmw-full} client. Complete the steps outlined in link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-60768C2F-72E1-42E0-8A17-CA76849F2950.html[Upgrade the Compatibility of a Virtual Machine Manually] ({vmw-full} documentation).
```

> . Convert the VM in the vSphere client from a VM to a template. Right-click on the VM and select **Template -> Convert to Template**. 

Reviewer: ```suggestion
. Convert the VM in the {vmw-short} client to a template by right-clicking on the VM and then selecting **Template -> Convert to Template**. 
```

+
[IMPORTANT]
====

> The steps for converting a VM to a template may change in future vSphere versions.

Reviewer: ```suggestion
The steps for converting a VM to a template might change in future {vmw-short} documentation versions.
```

![Uploading Screenshot from 2024-10-30 17-03-02.png…]()
